### ISSUE TITLE
Shimmer doesn't work properly on a ListView

### ISSUE THREAD

**Issue by haroonkhan9426** [hnvn/flutter_shimmer#19]
**What I want**
I want to shimmer a horizontal ListView.

**Issue**
All the widgets including shimmered and below shimmered, flows from left to right.

**Code**
```dart
Shimmer.fromColors(
      baseColor: Colors.grey[300],
      highlightColor: Colors.grey[100],
      child: ListView(
        scrollDirection: Axis.horizontal,
        children: <Widget>[
          Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
          Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
          Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
        ],
      ),
    );
```
**Output**
<img width="241" alt="Screenshot 2020-02-09 at 10 42 38 AM" src="https://user-images.githubusercontent.com/20193801/74097035-f76b4980-4b28-11ea-9ce2-8a5ba27b6a97.png">
<img width="240" alt="Screenshot 2020-02-09 at 10 42 54 AM" src="https://user-images.githubusercontent.com/20193801/74097037-fafed080-4b28-11ea-97f6-11f7a2ad7d36.png">

**Comment by haroonkhan9426** [comment]
I just overcame the issue but i think it's a bit more resource intensive way.
And just works perfectly.
```dart
ListView(
      scrollDirection: Axis.horizontal,
      children: <Widget>[
        Shimmer.fromColors(
          baseColor: Colors.grey[300],
          highlightColor: Colors.grey[100],
          child: Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
        ),
        Shimmer.fromColors(
          baseColor: Colors.grey[300],
          highlightColor: Colors.grey[100],
          child: Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
        ),
        Shimmer.fromColors(
          baseColor: Colors.grey[300],
          highlightColor: Colors.grey[100],
          child: Container(
            margin: EdgeInsets.all(4),
            width: 144,
            height: 203,
            decoration: BoxDecoration(
              borderRadius: BorderRadius.all(Radius.circular(10)),
              color: Colors.grey[400],
            ),
          ),
        ),
      ],
    );

**Comment by hnvn** [comment]
fixed in v1.1.0
